### ISSUE TITLE
Add Template Reference creates invalid custom parameters

### ISSUE THREAD

**Issue by RandomlyKnighted** [ligershark/side-waffle#345]
While working on the Aurelia templates for #273 I created a new template reference. Here is the parameters it created:

```xml
    <CustomParameters>
      <CustomParameter Name="SideWaffle Project Template Files" Value="Your name here" />
      <CustomParameter Name="8e0a6fff-7069-4384-9bb3-5e53d6cbb0b3" Value="f292d270-4e1c-416b-9434-1e3a6bb9b7a6" />
	  <CustomParameter Name="Item" Value="Item" />
    </CustomParameters>
```

It should actually look like this:

```xml
    <CustomParameters>
      <CustomParameter Name="$TemplateName$" Value="xUnit Test Project (DNX)" />
      <CustomParameter Name="$TemplateID$" Value="98669676-8c1b-4ccc-9602-3539b21e6380" />
      <CustomParameter Name="$TemplateType$" Value="Project" />
    </CustomParameters>
```

cc: @sayedihashimi

**Comment by sayedihashimi** [comment]
Good catch.

**Comment by RandomlyKnighted** [comment]
Where are we adding these elements at in SideWaffle?

**Comment by sayedihashimi** [comment]
The value for `$TemplateName$` and the others are getting replaced when the template is invoked. As you can see the info in the template corresponds to the `SideWaffle Project Template FIles` item template. To resolve it we should disable replacements in that file. You can do that in the .vstemplate file.

**Comment by sayedihashimi** [comment]
If are commented out I think it's fine. Let's see if anyone complains.

**Comment by RandomlyKnighted** [comment]
Alright, that works for me. Once we fix the problem with #350 then any PR that has this still commented out will get a failed Appveyor build. So I think that will handle any situation with contributing a new project template to SideWaffle.

I'll create and submit the PR this afternoon.
